docs/_data/nav.yml

```diff
@@ -6,9 +6,9 @@
   href: /support/
   category: support
 
-- title: Blog
-  href: /blog/
-  category: blog
+# - title: Blog
+#   href: /blog/
+#   category: blog
```

Comment:
Btw, I saw remove this instead of commenting out because we can easily revert this when we want a blog again and better not to have blog artifacts lying around until then.